### ISSUE TITLE
Discontinue parts of the project

### ISSUE THREAD

**Issue by lionirdeadman** [kurogetsusai/firefox-gnome-theme#62]
It seems that the project is missing people to work on it and as such, it seems that to keep the codebase working, it might be better to discontinue parts of it

Such as :

- Elementary CSD
- Gnome 3.16
- Gnome 3.26 (After 3.32 is released)
- Support for Firefox below 60 since that's the new ESR (and perhaps we should only support the latest firefox version, really)

**Comment by alizehctl** [comment]
I agree. With regards to elementary CSD, there is already another project maintaining a highly accurate elementary theme for Firefox https://github.com/Zonnev/elementaryos-firefox-theme

**Comment by rafaelmardojai** [comment]
I'm thinking about deprecating this repo and start a new one.
What do you thing guys?

**Comment by lionirdeadman** [comment]
What do you mean? @rafaelmardojai

**Comment by rafaelmardojai** [comment]
@lionirdeadman I'm firefox-gnome-theme current maintainer. Original author is not involved any more.

I had created a new repo: https://github.com/rafaelmardojai/firefox-gnome-theme. It will be a hard fork.
I'm reworking the theme structure and code.

So i'm going to deprecate this repo soon for the new one.

**Comment by lionirdeadman** [comment]
@rafaelmardojai Would it be worthwhile to join gnome-integration team instead? #9 

Also a link from this repo to whichever new one would be appropriate if no one plans to support this repo, is that fine @kurogetsusai?

**Comment by rafaelmardojai** [comment]
@kurogetsusai never wanted to join GNOME Integration Team so i will keep it like that.

She also stopped using GNOME and Firefox (https://github.com/kurogetsusai/firefox-gnome-theme/issues/52#issuecomment-438709654).
I do not think she has any problem if i put an advice on README.

**Comment by lionirdeadman** [comment]
My understanding is Kuro didn't see the point of joining GNOME Integration Team, not that they didn't want it. Also, if you're taking it and doing an hard fork, there's no reason why you couldn't do it but up to you, I guess.

But yeah, I doubt it'll bother him if you put advice in the README, just checking since it seems you've not talked to them about it.

**Comment by rafaelmardojai** [comment]
@lionirdeadman  You say talk with Kuro about it or with GNOME Integration Team?

**Comment by lionirdeadman** [comment]
Yes, I meant talk to Kuro about the advice in the readme.

GNOME Integration Team invited us back in #9, I doubt they'd say no but I guess a ping to @garrett won't hurt to see if they still want us in the Integration Team.

**Comment by rafaelmardojai** [comment]
Of course i will talk with her if finally i put an advice.

GNOME Integration Team looks pretty death. So i also doubt they'd say no if we're going to put some action there.
